### ISSUE TITLE
export makeViewClass as namespace?

### ISSUE THREAD

**Issue by grawk** [krakenjs/engine-munger#32]
There is a discrepancy in the README example which leads you to believe that the module exports `makeViewClass` as a named function:

``` js
var munger = require('engine-munger');
...
app.set("view", munger.makeViewClass(viewConfig));
```

Should the export statement be modified to match this example? Or should the example be changed to match the current behavior?

**Comment by aredridel** [comment]
Example to behavior! Good catch!

**Comment by aredridel** [comment]
Fixed in 3c470f495ba2f839fe840bf35f47585caf444c9d
